### ISSUE TITLE
Issue with k8s.io/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive/

### ISSUE THREAD

**Issue by dplacinta** [kubernetes/website#27407]
The terminal window has an error:
```shell
Kubernetes Bootcamp Terminal

Error Details: Environment not starting correctly.
```

**Comment by akshanshshrivastava** [comment]
any update on this issue I am still facing this
<img width="1505" alt="Screenshot 2022-02-22 at 9 34 44 PM" src="https://user-images.githubusercontent.com/21341620/155171388-ba1d70c7-3a96-40d7-9355-444bd5dafb1c.png">

**Comment by sftim** [comment]
That latest screenshot suggests an issue with Katacoda, rather than with the Kubernetes docs.
Check that you are not blocked by network filtering; if you're sure that you are not, you could contact Katacoda to report the issue.

**Comment by divya-mohan0209** [comment]
This issue could possibly be relevant to the ongoing [discussion](https://github.com/kubernetes/website/discussions/38878) on Katacoda removal.

**Comment by sftim** [comment]
We're [removing Katacoda](https://blog.k8s.io/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/). As such, we don't plan to fix this bug.

/triage unresolved

**Comment by tengqm** [comment]
/close

Won't fix because we're [removing Katacoda](https://blog.k8s.io/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/).
